Objects/longobject.c

```diff
@@ -4334,6 +4333,14 @@ long_pow(PyObject *v, PyObject *w, PyObject *x)
                 goto Error;
         }
 
+        /* if exponent < 0 and abs(modulus) == 1:
+               raise ValueError */
+        if ((Py_SIZE(b) < 0) && (c->ob_digit[0] == 1)) {
```

Comment:
Looks like this is only testing whether the least significant digit of the modulus is 1, not whether the entire modulus is 1.  Like the test below this, I expect this needs to also test for `Py_SIZE(c) == 1`.